ra/ra.go

```diff
@@ -168,6 +169,12 @@ func NewRegistrationAuthorityImpl(
 	}, []string{"limit", "decision"})
 	stats.MustRegister(rlCheckLatency)
 
+	overrideUsageGauge := prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "ratelimitsv1_override_usage",
+		Help: "Proportion of override limit used, by limit name and client identifier.",
+	}, []string{"limit", "client_id"})
```

Comment:
naming nit: I'd call it "override key" rather than "client identifier", because I think of "client" as being an ACME Client, and many of our override keys are domain names or other even weirder things.